### ISSUE TITLE
if helper causes view bindings to go off

### ISSUE THREAD

**Issue by kenips** [jasonmit/ember-auto-remove-jquery-events#1]
Not sure if other helpers will cause the same issue, but essentially with if helper the generated view's .$() is the same as the parent's, aka `this.$().is(this.get('parentView').$())` returns true. So the view bindings are unintentionally turned off.

**Comment by jasonmit** [comment]
Is that possible?  Can you give an example of how the parent and child share the same object?

**Comment by kenips** [comment]
hey @jasonmit I'm behind on 1.10 - just tested again and realized that it is now fixed in Ember 1.11 (but still broken in this library). Ember 1.11 throws the correct error:

```
Uncaught Error: Assertion Failed: You cannot access this.$() on a component with `tagName: ''` specified.
```
